docker-containers.el

```diff
@@ -218,7 +219,15 @@ Remove the volumes associated with the container when VOLUMES is set."
                                                                   (s-join " " ,(list (intern (format "docker-containers-%s-arguments" it))))))
              functions)))
 
-(docker-containers-create-selection-functions start stop restart pause unpause rm)
+(docker-containers-create-selection-functions
+  start
+  stop
+  restart
+  pause
+  unpause
+  rm
+  kill
+  )
```

Comment:
Dont put ending `)` on a line of its own please (https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html#Coding-Conventions)